kyaml/yaml/fns.go

```diff
@@ -180,12 +199,20 @@ type ElementMatcher struct {
 
 	// Create will create the Element if it is not found
 	Create *RNode `yaml:"create,omitempty"`
+
+	// NoValue indicates that matcher should only consider the key and ignore
+	// the actual value in the list. FieldValue must be empty when NoValue is
+	// set to true.
+	NoValue bool `yaml:"noValue,omitempty"`
```

Comment:
The field name `MatchAnyValue` would be clearer here.

It's like using a wildcard `*` in the value.

Explicitly supporting wildcards or more general regexps could be the next step.

Any opinion on adding  `MatchAnyField`?

Reply:
Yeah, `MatchAnyValue` is better.